packages/i18n/src/locales/en.i18n.json

```diff
@@ -1125,8 +1126,8 @@
   "Common_Access": "Common Access",
   "Commit": "Commit",
   "Community": "Community",
-  "Contextualbar_resizable": "Contextual bar resizable",
-  "Contextualbar_resizable_description": "Allows you to adjust the size of the contextual bar by simply dragging, giving you instant customization and flexibility",
+  "Contextualbar_resizable": "Resizable contextual bar",
```

Comment:
You can't change translations without changin the keys. You have to either remove this key and create a new one, or you have to delete all keys from the other languages, so that the strings get translated again.

Only cases you can edit translations is if you need to fix something like a typo that does not change it's meaning or any terms that would affect other translations.

Reply:
I've checked the other i18n languages, but it doesn't seem they have these keys defined yet...